recipes/devices/families/pine64.sh

```diff
@@ -7,6 +7,7 @@
 BASE="Debian"
 ARCH="armhf"
 BUILD="armv7"
+UINITRD_ARCH="arm"
```

Comment:
This should be arm64, boot.scr needs to be compiled as arm

Reply:
Fixed in 2dd9b13